content/guides/antora.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Antora
+description: Build your website with the Antora Static Site Generator (SSG) and host it on Clever Cloud. No dedicated runner needed.
+tags:
+- guides
+keywords:
+- antora
+- asciidoc
+- nodejs
+- static
+- ssg
+- html
+- js
+- css
+- website
+type: "docs"
+comments: false
+draft: false
+---
+[Antora](https://antora.org/) is an [AsciiDoc](https://asciidoc.org/) based Static Site Generator focused on technical documentation writing. It promotes *docs as code*, using a YAML playbook and git repositories to organize and host content. If you need an example source code, use the [demo playbook](https://docs.antora.org/antora/latest/install-and-run-quickstart/#create-a-playbook) (you'll need [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Node.js](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs)):
+```bash
+git init
+npm i -D -E @antora/cli @antora/site-generator
+echo -e "site:\n  title: Antora Docs\n  start_page: component-b::index.adoc\ncontent:\n  sources:\n  - url: https://gitlab.com/antora/demo/demo-component-a.git\n    branches: HEAD\n  - url: https://gitlab.com/antora/demo/demo-component-b.git\n    branches: [v2.0, v1.0]\n    start_path: docs\nui:\n  bundle:\n    url: https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/HEAD/raw/build/ui-bundle.zip?job=bundle-stable\n    snapshot: true" > antora-playbook.yml
+
+```
```

Comment:
- what are the flags you're using and why to you use them?
- what exactly is doing this script?

Reply:
`-D` is to use packages as dev dependencies. `-E` can be removed here as I remove the exact version (but we can also use latest version in this command if you prefer).

The `echo` line is to create the `playbook.yml` demo file which will be used to deploy